docs/README.md

```diff
@@ -1,8 +1,15 @@
-# Summary
+---
+id: index
+title: textlint Documentation
```

Comment:
```diff
- title: textlint Documentation
+ title: Documentation
```

Because, 
https://deploy-preview-468--textlint.netlify.com/docs/index.html `<title>textlint Documentation · textlint</title>`

Reply:
Got your point. Will remove redundant "textlint" from title.